dev-packages/application-manager/src/application-package-manager.ts

```diff
@@ -78,7 +78,21 @@ export class ApplicationPackageManager {
     }
 
     startElectron(args: string[]): cp.ChildProcess {
-        const { mainArgs, options } = this.adjustArgs([this.pck.frontend('electron-main.js'), ...args]);
+        // If possible, pass the project root directory to electron rather than the script file so that Electron
+        // can determine the app name. This requires that the package.json has a main field.
+        let entryPoint = this.pck.projectPath;
```

Comment:
`entryPoint = this.pck.pck.main`?

Reply:
This should be `projectPath` actually. I've renamed this variable to `appPath` to make it consistent with the electron documentation:

```
Electron 4.2.12 - Build cross platform desktop apps with JavaScript, HTML, and CSS
Usage: electron [options] [path]

A path to an Electron app may be specified. It must be one of the following:
  - index.js file.
  - Folder containing a package.json file.
  - Folder containing an index.js file.
  - .html/.htm file.
  - http://, https://, or file:// URL.
```

This variable is the "path to an Electron app" argument.